orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/servergroup/TitusInterestingHealthProviderNamesSupplier.java

```diff
@@ -34,8 +36,8 @@
 public class TitusInterestingHealthProviderNamesSupplier implements InterestingHealthProviderNamesSupplier {
   private static final String TITUS = "titus";
   private static final String INTERESTING_HEALTH_PROVIDER_NAMES = "interestingHealthProviderNames";
-  private static final String ROLLING_PUSH = "rollingpush";
-  private static final String SUPPORTED_STAGE = "rollingPush";
+  private static final List<String> SUPPORTED_STRATEGIES = Collections.singletonList("rollingpush");
```

Comment:
How important is it to restrict only to `rollingpush` and are there any negative side effects if this were applied to _any_ titus clone/create (regardless of strategy)?

ie) If I did a red/black of a titus asg that had an interesting health provider names label on it... should the red/black adhere to those health provider names?

Just asking because it looks like it would work out of the box if the `rollingpush` check was removed.